book-es/gen_cover.py

```
@@ -0,0 +1,66 @@
+        print("Cover page will use default TikZ vector art in cover.tex.")
+        return False
+    from openai import OpenAI
+    import base64, urllib.request
+    client = OpenAI()
+    try:
+        r = client.images.generate(model="gpt-image-1", prompt=prompt,
+                                    size="1024x1536", quality="high", n=1)
+        data = base64.b64decode(r.data[0].b64_json)
+        open(out, "wb").write(data)
+        return True
+    except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)